tests/test_quadtree.py

```diff
@@ -39,16 +48,20 @@ def test_str_with_particles(self):
         self.assertEqual(actual, expected)
 
     def test_query(self):
+        particle_num = 5
```

Comment:
my autism says this should be renamed from particle_num to num_particles

Reply:
Valid